tests/lib/test_io.py

```diff
@@ -0,0 +1,203 @@
+# pylint: disable=missing-function-docstring disable=missing-module-docstring
```

Comment:
I think missing-function-docstring and missing-module-docstring should not be disabled but fixed

Reply:
Or the rules should be updated

Reply:
Yes but the issue is that it seems that with pylint, there is no way to re-declare a partial .pylintrc for a dedicated folder such as with eslint.